From d26bdf641ac2647a4d2d0a2fbafe1ac649e6f189 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Fri, 6 Sep 2013 19:57:30 +0800
Subject: [PATCH 0175/1074] ENGR00278646-5 ARM: dts: imx6qdl-sabresd: add
 no-1-8-v property for usdhc

The sabresd board does not have external 1.8v/3.3v signal voltage switch
support for usdhc.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index be5829f..c96b806 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -523,6 +523,7 @@
 	bus-width = <8>;
 	cd-gpios = <&gpio2 2 0>;
 	wp-gpios = <&gpio2 3 0>;
+	no-1-8-v;
 	status = "okay";
 };
 
@@ -532,6 +533,7 @@
 	bus-width = <8>;
 	cd-gpios = <&gpio2 0 0>;
 	wp-gpios = <&gpio2 1 0>;
+	no-1-8-v;
 	status = "okay";
 };
 
-- 
1.7.5.4

